app/views/layouts/application.html.haml

```diff
@@ -14,6 +14,7 @@
       window.BikeIndex.translator = (keyspace) => {
         return (key, args={}) => I18n.t(`javascript.${keyspace}.${key}`, args);
       }
+    = yield :header
```

Comment:
Could you add a comment in here about why this is here? (e.g. `see news/show for example of content_for :header`)

I think this is the correct way to solve this problem, but I worry that coming back to this in a few months, it will be unclear why this yield is here.

Reply:
No prob!